ts/adaptors/linkedomAdaptor.ts

```diff
@@ -0,0 +1,148 @@
+    '\u3250-\u4DBF', // Enclosed CJK Letters and Months ... CJK Unified Ideographs Extension A
+    '\u4E00-\uA4C6', // CJK Unified Ideographs ... Yi Radicals
+    '\uA960-\uA97C', // Hangul Jamo Extended-A
+    '\uAC00-\uD7A3', // Hangul Syllables
+    '\uF900-\uFAFF', // CJK Compatibility Ideographs
+    '\uFE10-\uFE19', // Vertical Forms
+    '\uFE30-\uFE6B', // CJK Compatibility Forms ... Small Form Variants
+    '\uFF01-\uFF60\uFFE0-\uFFE6', // Halfwidth and Fullwidth Forms
+    '\u{1B000}-\u{1B001}', // Kana Supplement
+    '\u{1F200}-\u{1F251}', // Enclosed Ideographic Supplement
+    '\u{20000}-\u{3FFFD}', // CJK Unified Ideographs Extension B ... Tertiary Ideographic Plane
+    ']'
+  ].join(''), 'gu');
+
+  /**
+   * The options for the instance
+   */
+  public options: OptionList;
+
+  /**
+   * @param {Window} window   The window to work with
+   * @param {OptionList} options  The options for the jsdom adaptor
+   * @constructor
+   */
+  constructor(window: Window, options: OptionList = null) {
+    super(window);
+    window.constructor.prototype.HTMLCollection = class {};  // add fake class for missing HTMLCollecton
+    let CLASS = this.constructor as typeof LinkedomAdaptor;
+    this.options = userOptions(defaultOptions({}, CLASS.OPTIONS), options);
+  }
+
+  /**
+   * @override
+   */
+  public parse(text: string, format: string = 'text/html') {
+    if (!text.match(/^(?:\s|\n)*</)) text = '<html>' + text + '</html>';
+    return this.parser.parseFromString(text, format);
+  }
+
+  /**
+   * @override
+   *
+   * This will do an HTML serialization, which may be good enough, but
+   *   won't necessarily close some tags properly.
+   */
+  public serializeXML(node: HTMLElement) {
+    return this.outerHTML(node);
+  }
+
+  /**
+   * Linkedom doesn't implement getComputedStyle(), so just
+   * use the default value (unfortunately).
+   *
+   * @override
+   */
+  public fontSize(_node: HTMLElement) {
+    return this.options.fontSize;
+  }
+
+  /**
+   * Linkedom doesn't implement getComputedStyle(), so just
+   * use the default value (unfortunately).
+   *
+   * @override
+   */
+  public fontFamily(_node: HTMLElement) {
+    return this.options.fontFamily;
+  }
+
+  /**
+   * @override
+   */
+  public nodeSize(node: HTMLElement, _em: number = 1, _local: boolean = null) {
+    const text = this.textContent(node);
+    const non = Array.from(text.replace(LinkedomAdaptor.cjkPattern, '')).length; // # of non-CJK chars
+    const CJK = Array.from(text).length - non;                                   // # of cjk chars
+    return [
+      CJK * this.options.cjkCharWidth + non * this.options.unknownCharWidth,
+      this.options.unknownCharHeight
+    ] as [number, number];
+  }
+
+  /**
+   * @override
+   */
+  public nodeBBox(_node: HTMLElement) {
+    return {left: 0, right: 0, top: 0, bottom: 0};
+  }
+
+}
+
+/**
+ * Function for creating an HTML adaptor using jsdom
```

Comment:
jsdom -> linkedom

Reply:
Thanks, I've made a commit that fixes this, and the other location where jsdom was accidentally left in.